groups/instance/profiles/shared-services-eu-west-2/devops/vars

```diff
@@ -0,0 +1,6 @@
+account_name         = "shared-services"
+environment          = "devops"
+region               = "eu-west-2"
+secrets_account_name = "development"
+secrets_environment  = "devops1"
```

Comment:
We shouldn't be using secrets created for a different environment/deployment as it could lead to unintended consequences if updates are made / old secrets are removed.

We should instead create a new set of secrets specific for this deployment in the appropriate Vault path.

Reply:
Implemented via commit: 252d107678658864551f07a8c8958fea666199bc